includes/class-logger.php

```diff
@@ -45,21 +45,7 @@ class Logger {
 	 *     'debug': Debug-level messages.
 	 */
 	public static function log( $message, $level = 'info' ) {
-		if ( ! self::can_log() ) {
-			return;
-		}
-
-		self::init_logger();
-		self::$logger->log( $level, $message, [ 'source' => self::LOG_FILENAME ] );
-	}
-
-	/**
-	 * Initiate logger property with the WooCommerce core logger only if it's not set already
-	 */
-	public static function init_logger() {
-		if ( ! isset( self::$logger ) && ! is_object( self::$logger ) ) {
-			self::$logger = wc_get_logger();
-		}
+		wcpay_get_container()->get( \WCPay\Internal\Logger::class )->log( $message );
```

Comment:
Could you please add a `use` statement for the `Logger` class?

Reply:
Done